app/models/pay/subscription.rb

```diff
@@ -12,6 +12,8 @@ class Subscription < Pay::ApplicationRecord
     scope :cancelled, -> { where.not(ends_at: nil) }
     scope :on_grace_period, -> { cancelled.where("#{table_name}.ends_at > ?", Time.zone.now) }
     scope :active, -> { where(status: ["trialing", "active"], ends_at: nil).or(on_grace_period).or(on_trial) }
+    scope :not_paused, -> { where("data->>'pause_behavior' IS NULL") }
```

Comment:
This scope probably needs to build upon the `active` scope since `paused` subscriptions are active ones.

```ruby
    scope :not_paused, -> { active.where("data->>'pause_behavior' IS NULL") }
```

`active_without_paused` maybe?